### DISCUSSION TITLE
[WASM] <-> [WindowsAppSdk] How 'pixel perfect' is UNO?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#8276] [Q&A]
I was working off the assumption that I would experience virtually the same size of UIs on both WindowsAppSdk and WASM. However, I found this not being accurate. Please see attached screenshot (MS Word vertical ruler in the background on the left side), where I compared the same test UI. This is latest Chrome (100% zoom) on Windows 11.

What am I doing wrong?

![image](https://user-images.githubusercontent.com/26179242/157285108-c076e617-600c-4f08-b161-ecab9538313d.png)
![image](https://user-images.githubusercontent.com/26179242/157285162-1bf13f7e-763f-48cc-8461-5a3c929d85a8.png)

**Comment by jeromelaban** [comment]
Thanks for the report. Can you point to the differences you're seeing? Note that default fonts are likely different here, changing the layout.

**Reply by DierkDroth** [reply]
Thanks for follow up @jeromelaban.

I'm referring e.g. to the height of controls: when comparing the total height of all rows - and measuring them with MS Word ruler visible on the left side - you will notice that the WASM UI is substantially higher (~14.25") than the WindowsAppSDK UI (~12.5").

I'm using [Google Roboto font, Regular 400](https://fonts.google.com/specimen/Roboto) imported for WASM - for WindowsAppSdk it appears to be available without further ado. To import I followed the instructions [here ](https://platform.uno/docs/articles/features/custom-fonts.html)and [here](https://blog.mzikmund.com/2020/01/custom-fonts-in-uno-platform/).

Please let me know if you needed any further details. Thanks

**Reply by DierkDroth** [reply]
@jeromelaban this issue is crucial, since it impact´s the way how I needed to move forward with my marketing team. Could you confirm and/or reproduce the issue? What would be the options to address this issue?

**Reply by jeromelaban** [reply]
Thanks, no we cannot reproduce these issue specifically. If you have specifics, please open an issue with a repro that demonstrates individual differences.

**Reply by DierkDroth** [reply]
Interesting: UNO WASM visualization on Chrome and Edge is the same size (guess expected), but on FireFox it´s actually smaller

**Reply by DierkDroth** [reply]
@jeromelaban please see attached. This is a simplified version of the test in my original post above - I left my custom theming to remove variables. This is just UNO standard theme.

I also added screenshot for WindowsAppSdk and WASM/Chrome

[Scaling.zip](https://github.com/unoplatform/uno/files/8233551/Scaling.zip)

![image](https://user-images.githubusercontent.com/26179242/157906646-20c158ba-e56f-48f5-8177-022abbacb81d.png)
![image](https://user-images.githubusercontent.com/26179242/157906690-dd4624ca-6e96-4e5b-9ada-f1c95a604fa0.png)

**Reply by jeromelaban** [reply]
Thanks, let's track this in https://github.com/unoplatform/uno/issues/8310
